tv_apps/browser/manifest.webapp

```diff
@@ -1,61 +1,210 @@
+      "description": "Browser"
+    },
+    "et-EE": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "fi-FI": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "fr-CA": {
+      "name": "Browser",
+      "description": "Browser"
     },
-    "fr": {
-      "name": "Navigateur",
-      "description": "Navigateur Web Gaia"
+    "fr-FR": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "hi": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "hr-HR": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "hu-HU": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "id": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "it-IT": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "ja-JP": {
+      "name": "ブラウザ",
+      "description": "ブラウザ"
+    },
+    "lt-LT": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "lv-LV": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "nl-NL": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "no-NO": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "pl-PL": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "pt-BR": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "pt-PT": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "ro-RO": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "ru-RU": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "ru": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "sk-SK": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "sl-SI": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "sr-YU": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "sv-SE": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "th": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "tr-TR": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "vi": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "zh-CN": {
+      "name": "Browser",
+      "description": "Browser"
+    },
+    "zh-HK": {
+      "name": "Browser",
+      "description": "Browser"
     },
     "zh-TW": {
-      "name": "網路瀏覽器",
-      "description": "Gaia 網路瀏覽器"
+      "name": "Browser",
+      "description": "Browser"
     }
   },
   "default_locale": "en-US",
   "icons": {
-    "84": "/shared/resources/branding/browser_84.png",
-    "126": "/shared/resources/branding/browser_126.png",
-    "142": "/shared/resources/branding/browser_142.png",
-    "189": "/shared/resources/branding/browser_189.png",
-    "284": "/shared/resources/branding/browser_284.png"
+    "336": "/style/icons/icon.png"
```

Comment:
Thanks! Use 284 pixel icon temporarily. Track this in Bug 1196624 - Need official and unofficial 336 pixel Firefox branding icon for Firefox OS TV .